phpstan.neon

```diff
@@ -28,7 +28,6 @@ parameters:
                 power_shell: bool,
                 user_id: int,
                 root_dir: string,
-                env: string,
```

Comment:
Removed in https://github.com/jolicode/docker-starter/commit/f4957592cb41f70f9fa45325548426dbc8662edd